package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ejs-loader",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Please remove this from PR since I’m using a script to automate this.

Reply:
Ok